test/specs/code/expression_test.dart

```diff
@@ -141,6 +142,31 @@ void main() {
     );
   });
 
+  test('should emit a record', () {
+    expect(literalRecord([], {'named': 1}), equalsDart('(named: 1)'));
+  });
+
+  test('should emit a const record', () {
+    expect(
+        literalConstRecord([], {'named': 1}), equalsDart('const (named: 1)'));
+  });
+
+  test('should emit a record of other literals and expressions', () {
+    expect(
+        literalRecord([
+          1,
+          refer('one'),
+          'one'
+        ], {
+          'named': refer('Foo').newInstance([literalNum(1)])
+        }),
+        equalsDart("(1, one, 'one', named: Foo(1))"));
+  });
+
+  test('should correctly emit a record with a single positional field', () {
```

Comment:
Great tests, can I ask for two more, just because of the delicate commas:

* empty record, which is allowed (`()`)
* record with only named fields

Reply:
Sure! I added an empty record. There already was a test for only named fields, but I added/renamed some tests to make it clearer that we test all three cases of only positional, only named and mixed.